app/src/main/java/com/example/anfcodetest/CustomViewHolder.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.anfcodetest
+
+import android.content.Intent
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import android.view.View
+import android.widget.Button
+import androidx.core.content.ContextCompat.startActivity
+import androidx.recyclerview.widget.RecyclerView
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.card_row.view.*
+
+class CustomViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
+
+    fun bind(card: Card) = with(view) {
```

Comment:
For this bind, you can create separate methods for handling the different pieces of data. I'm sure the other guys will have something cleaner for this, but this gets the general idea across. You can do the checks for nulls/empty strings in these methods, and clear up this main bind method so it's not so daunting (yup, this also makes it easier to test and debug!)